docs/usage/config-validation.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Config Validation
+description: How to validate Renovate's configuration.
+---
+
+# Config Validation
+
+All [`renovate` distributions](getting-started/running.md#available-distributions) contain a standalone validator program (`renovate-config-validator`) that can be used to validate Renovate's configuration.
+
+The validator program checks files passed as CLI arguments.
+If no argument is given, all [default locations](../configuration-options.md) (if files exist) and the `RENOVATE_CONFIG_FILE` environment variable are checked.
+
+```shell-session
```

Comment:
Rendered like this:

<img width="507" alt="image" src="https://user-images.githubusercontent.com/1540469/236438182-63a71d48-a83c-4842-9234-a2e529ef615a.png">

Reply:
That's very cool! I did not know we could do that! 🥳

Reply:
what about using `console` as language keyword?

Reply:
Do console and shell-session render the same?

Reply:
Yes!

Reply:
Let's use `console` then, it's shorter and easier to remember.

Reply:
Sure, done!